### ISSUE TITLE
please provide a tutorial

### ISSUE THREAD

**Issue by FabioPedretti** [lincity-ng/lincity-ng#114]
It would be nice to have a little tutorial to ease learning the game and its mechanics.
Something like the tutorial in the game 0 A.D..

Thanks! ️🙂

**Comment by dbear496** [comment]
I fully agree that a decent tutorial for Lincity-NG would be very helpful for newer players. Some of the game mechanics are very unintuitive and need some explanation.

If there is a particular feature or layout that you would like in a tutorial, then please be specific. Simply saying you want it to look like another game is too general. I have never played nor plan to ever play 0AD, but even if I did, I would likely have different takeaways than you.

**Comment by dbear496** [comment]
That is an interesting idea. It sounds like an interactive guide on how to use the UI. This is a very low priority in my book since I think the UI is already pretty intuitive. Nevertheless, I am open to PRs.
